recipes/kdalgorithms/all/test_package/main.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+  This file is part of KDAlgorithms.
+
+  SPDX-FileCopyrightText: 2025 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+
+  SPDX-License-Identifier: MIT
+
+  Contact KDAB at <info@kdab.com> for commercial licensing options.
+*/
+
+#include <iostream>
+#include <kdalgorithms.h>
+#include <string>
+#include <vector>
+
+int main()
+{
+    auto vec = kdalgorithms::iota(1, 100);
+    auto odds = kdalgorithms::filtered(vec, [](int i) { return i % 2 == 1; });
+    auto result = kdalgorithms::accumulate(odds, [](const std::string &partial, int value) {
+        return partial + "," + std::to_string(value);
+    });
+
+    std::cout << result << "\n";
+
+    return 0;
+}
```

Comment:
```suggestion
#include <iostream>
#include <kdalgorithms.h>

int main()
{
    auto vec = kdalgorithms::iota(1, 10);
    std::cout << "kdalgorithms::iota: " << vec[0] << "\n";

    return 0;
}
```

The test package is mainly to validate the package only: if we can link/include header without problems. So we try to keep it as simples as possible.

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/folders_and_files.md#test_package